fineract-provider/src/main/resources/db/changelog/tenant/parts/0052_additional_flag_loan_repayment_schedule.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wrong liquibase changeset!
0052_loan_transaction_chargeback.xml is missing!

Reply:
Done